### ISSUE TITLE
Binary classification using MLP

### ISSUE THREAD

**Issue by omiderfanmanesh** [kekmodel/MPL-pytorch#32]
Hi 
Thank you for your valuable repository. How to change the loss function for binary classification?
